protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -63,7 +63,15 @@ service TradingDataService {
     // Positions
     //
     // Get a list of positions by party (public key) using cursor based pagination
-    rpc ListPositions(ListPositionsRequest) returns (ListPositionsResponse);
+    // Deprecated: use ListAllPositions instead
+    rpc ListPositions(ListPositionsRequest) returns (ListPositionsResponse) {
+        option deprecated = true;
+    };
+
+    // Positions
+    //
+    // Get a list of positions by party (public key) using cursor based pagination
+    rpc ListAllPositions(ListAllPositionsRequest) returns (ListAllPositionsResponse);
 
     // Positions subscription
```

Comment:
Is everything under "Positions subscription" deprecated? If so - could you add (deprecated) to this bit of text, and can you add a title - as was done here - to the replacement?

Reply:
No, positions subscription is not deprecated, it has not been affected and should continue to work as it does.

Reply:
Thanks for clarifying! Hard to tell in the protos formatting sometimes